### ISSUE TITLE
[nsis] Possibility to edit installer name ?

### ISSUE THREAD

**Issue by RilRil** [electron-userland/electron-builder#1221]
* **Version**: 13.0.0

* **Target**: win32 

Hello, 
When using NSIS, is it possible to change the name of the installer ?
I explain. Here is a piece of my package.json : 
```
{
 "name": "myapp",
 "productName": "My App",
 "version": "1.0.0",
 }
```
the installer created is named : "My App Setup 1.0.0.exe". I dont want to put this version number in the name as it is a private version number..
Is there a way to accomplish this ?

thanks for your help !

**Comment by develar** [comment]
Implemented in upcoming 13.2.0: option `nsis.artifactName` is added.

**Comment by RilRil** [comment]
awesome. it is exactly what I needed. Thanks for the great work!
